client/src/components/Perk.svelte

```diff
@@ -0,0 +1,55 @@
+<script>
+    import { onMount } from 'svelte';
+
+    export let buttonText = 'View Perk';
+
+    let text = 'Perk Loading...';
+
+    onMount(async () => {
+        const response = await fetch('/api/dynamic-text');
```

Comment:
Reminder to change fetch location once the PerkController is set up. It should fetch the specific Perk from the PerkRepository